### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MeepMoop**
Added C# cube reps, fixed c++ link.

### CODE REVIEW

**Comment by justinj** [comment]
Thanks! :thumbsup:

File: README.md [hunk 1]
```diff
@@ -40,10 +41,16 @@ brief description of what it does.
 * [cubeseer](https://github.com/justinj/cubeseer) - Diagram generator, but contains a cube representation
 * [RaMSDeL](https://github.com/justinj/ramsdel) - Random Move Scrambler Definition Language
 
-#C++
-
+C++
+===
```
Comment (jfly) [inline README.md:45]: Why the inconsistency with the other language headers?

Comment (justinj) [inline README.md:45]: The C# one needed some weird escaping for the # to show up using the other way, so it's more readable like this. I guess the right thing would be to change all the titles to use this format.

Comment (jfly) [inline README.md:45]: Ah, I see. I think I'd prefer the escaping, but your call. Either way, does the c++ need this?

Comment (justinj) [inline README.md:45]: I don't really care too much, I already changed it to all use the `===` way, but I don't mind changing it back. C++ needed the explicit link to get the linking from the top to work.

Comment (justinj) [inline README.md:45]: Changed back to `#` headers in 29a1fed

**Pull request merged**